build/common.ps1

```diff
@@ -189,12 +190,34 @@ Function Install-NuGet {
     & $NuGetExe locals all -list -verbosity detailed
 }
 
+Function Get-MsBuildPath{
+    [CmdletBinding()]
+    param()
+
+    if (-Not (Test-Path $VsWhereExe))
+    {
+        throw "Expected VsWhere.exe at '$VsWhereExe' but it was not present. This is needed to locate MSBuild.exe"
+    }
+
+    $path = & $VsWhereExe -latest -products * -requires Microsoft.Component.MSBuild -property installationPath
+    if ($path) {
+        $path = join-path $path 'MSBuild\15.0\Bin\MSBuild.exe'
+        if (test-path $path) {
+        return $path
+        }
+    }
+    
+    throw "Unable to locate MSBuild using VsWhere.exe"
+}
+
 Function Install-DotnetCLI {
     [CmdletBinding()]
     param(
         [switch]$Force
     )
-    $msbuildExe = 'C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\MSBuild\15.0\bin\msbuild.exe'
+
+
+    $msbuildExe = Get-MsBuildPath 
```

Comment:
Just looking at the scripts for an unrelated task. 

Doesn't using Get-MSBuildExe fix it already?

Reply:
im not following what you are asking about this

Reply:
Instead of using the VSWhere logic to find MSBUild, does simply using that function solve the problem? 
Get-MSBuildExe is already something that exists and a quick look suggests that it might handle the VS on a different disk scenario.

Reply:
No, that thing that assumes its on the drive with program files x86.  I dont even have the VS15COMNTOOLS environment variable its talking about, just the ones for 12.0 and 14,.0,

For completeness it should probably be changed also.